test/auth.test.js

```diff
@@ -4,11 +4,11 @@ const { defaultSdk, mock } = require('./utils.js');
 const resources = require('./resources/auth.json');
 const { isConnectionError } = require('../lib/util');
 
-describe('auth', () => {
-    afterEach(() => {
+describe('auth', function () {
```

Comment:
why move away from arrow functions?

Reply:
and why only in tests? maybe bad eslint settings that you had temporarily? cannot see anything in the config

Reply:
its a slight preference of eslint + prettier due to some formatting issues which can exist. I just added test folder to lint:fix and it replaced

Reply:
uhm, i get that, but it's not used in the rest of the code hence I'm sensing some kind of bad config?